tests/src/test/java/alluxio/proxy/s3/S3ClientRestApiTest.java

```diff
@@ -196,33 +203,86 @@ public void deleteNonEmptyBucket() throws Exception {
 
     try {
       // Delete a non-empty bucket should fail.
-      new TestCase(mHostname, mPort, S3_SERVICE_PREFIX + AlluxioURI.SEPARATOR + bucketName,
-          NO_PARAMS, HttpMethod.DELETE, null, TestCaseOptions.defaults()).run();
-      Assert.fail("delete a non-empty bucket should fail");
+      deleteBucketRestCall(bucketName);
+    } catch (AssertionError e) {
+      // expected
+      return;
+    }
+    Assert.fail("delete a non-empty bucket should fail");
+  }
+
+  @Test
+  public void putObject() throws Exception {
```

Comment:
can we add another test case for object with a larger size , say 8M ?

Reply:
done